File: AICodingWeb/Models/SupplierCategory.cs
using System;
using System.Collections.Generic;

namespace AICodingWeb.Models
{
    public class SupplierCategory
    {
        public int SupplierCategoryID { get; set; }
        public string SupplierCategoryName { get; set; }
        public int LastEditedBy { get; set; }
        public DateTime LastEditedWhen { get; set; }

        public virtual ICollection<Supplier> Suppliers { get; set; }
    }
} 

File: AICodingWeb/aicoding_web/app/models/supplier_category.py
from datetime import datetime
from app import db
from app.models.base import BaseModel

class SupplierCategory(db.Model, BaseModel):
    """SupplierCategory model"""
    
    __tablename__ = 'supplier_categories'
    
    SupplierCategoryID = db.Column(db.Integer, primary_key=True)
    SupplierCategoryName = db.Column(db.String(50), nullable=False)
    LastEditedBy = db.Column(db.Integer, db.ForeignKey('people.PersonID'), nullable=False)
    ValidFrom = db.Column(db.DateTime, nullable=False)
    ValidTo = db.Column(db.DateTime, nullable=False)
    
    def __repr__(self):
        return f'<SupplierCategory {self.SupplierCategoryID}>' 